Makefile

```diff
@@ -18,6 +18,9 @@ $(INSTALL_STAMP): $(PYTHON)
 	$(VENV)/bin/pip install -Ue .
 	touch $(INSTALL_STAMP)
 
+install-postgres: $(INSTALL_STAMP)
+	$(VENV)/bin/pip install -Ue .[postgresql]
```

Comment:
This does not install postgresql deps in Travis
https://travis-ci.org/mozilla-services/kinto/jobs/69301205#L244